### PR TITLE
whitepaperSection

### PR DESCRIPTION
**Pull request by ZNicoZ**

### CODE REVIEW

File: content/whitepaper.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Whitepaper
+slug: wp
+---
+
+## Get a free copy of our whitepapers, no need for a registration - just go ahead:
```
Comment (svettwer) [inline content/whitepaper.md]:
```suggestion
## Get a copy of our whitepapers for free
```

File: content/whitepaper.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+{{<whitepaper "documents/Dos_Donts_WP_Test.pdf""10 Dos and Don´ts of E2E Monitoring""far fa-file-pdf">}}
```
Comment (svettwer) [inline content/whitepaper.md]: Link is not working. Currently leads to a 404.

**Comment by svettwer** [comment]
I think it would be great to open the download in a new tab with `_blank`. Then the user could read the PDF and switch back to the website for further Sakuli awesomeness. =)

File: themes/sakuli/layouts/shortcodes/whitepaper.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<div
+    style="background-color: #fff; padding: 20px; box-shadow: 0px 0px 10px #7C7C7B; margin-left: 0%; margin-right: 0%; margin-top: 20px; margin-bottom: 20px">
+            <div class="flexedContainer">
+                <div class="preview__item">
+                    <a href="{{ .Get 0 }}"><img src="/images/wp/{{ .Get 2 }}" alt="preview document" class="preview__img"> </a>
```
Comment (svettwer) [inline themes/sakuli/layouts/shortcodes/whitepaper.html]:
```suggestion
                    <a href="{{ .Get 0 }}" target="_blank"><img src="/images/wp/{{ .Get 2 }}" alt="preview document" class="preview__img"> </a>
```

**Comment by cliffle** [comment]
I noticed that when I hover the mouse over the download icon a little bar shows up in front of  `10 Dos and Don´ts of E2E Monitoring`.

**Comment by ZNicoZ** [comment]
add _blank / _rel

**Pull request merged**